apps/condo/domains/banking/schema/PredictTransactionClassificationService.js

```diff
@@ -102,6 +109,9 @@ const PredictTransactionClassificationService = new GQLCustomSchema('PredictTran
                 if (!costItem) {
                     throw new GQLError(errors.COST_ITEM_NOT_FOUND, context)
                 }
+                if (costItem.isOutcome !== isOutcome) {
+                    throw new GQLError(errors.COST_ITEM_IS_OUTCOME_NOT_EQUAL, context)
```

Comment:
Please, attach information about purpose and costItem, that was wrongly detected.
You can use error message interpolation, that i've made in GQLError.